cfpb/cfgov-refresh
# clean up whitespace + formatting for success notification html

Whitespace + styling improvements for the conference registration form success notification.

## Additions

- Adds a `block` element around the success message for the conference registration form so that the whitespace below is increased

## Changes
- Changes `h4` element for the message inside the success notification template to a `div` element - this allows for rich text fields to output child content inside the message container. This won't affect styling in other notifications elsewhere, because the `.h4` class is still applied to the `div` element.

## Testing

- check out locally
- Create a 'conference registration form' element in the Wagtail admin
- Add some text to the "Success message" field - here's mine:
  > Thank you for registering for the CFPB research conference on December 15–16, 2016. We are looking forward to seeing you there!
  > Let us know if your plans change, and you can’t make the days you registered to attend. Also, let us know if you have any questions about the event and how to attend. Feel free to email us at CFPB_ResearchConference@cfpb.gov.
- View page, submit form, and see that the green notification message appears formatted as expected

## Review

- @chosak 
- @nataliafitzgerald 


## Screenshots

![screen shot 2016-11-21 at 12 52 06 pm](https://cloud.githubusercontent.com/assets/702526/20496067/abd30b78-aff1-11e6-9f78-5c304dd5319e.png)

![screen shot 2016-11-21 at 1 52 16 pm](https://cloud.githubusercontent.com/assets/702526/20496096/c4c26c78-aff1-11e6-8268-413011305c87.png)



## Notes

-

## Todos

-

## Checklist
